general/1_terminal/Bogdana_Maksiuta/task04.txt

```diff
@@ -0,0 +1,2 @@
+cat ./result.json | jq '.messages[] .text'
```

Comment:
As discussed, this answer is accepted. If you'd like an additional assignment, you may try to make it look prettier, i.e. getting rid of arrays.